src/components/ReservedMissions.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './styles/reserved.css';
+
+const ReservedMissions = () => {
+  const { missions } = useSelector((state) => state.missions);
+  const reservedMissions = missions.filter((list) => list.reserved === true);
+
+  return (
+    <sections>
+      <h2>My Missions</h2>
+      <ul className="mission-list">
+        {reservedMissions.map((mission) => {
+          const { id, name } = mission;
+          return <li key={id} className="mission-name">{name}</li>;
+        })}
+      </ul>
+    </sections>
+  );
+};
+
+export default ReservedMissions;
```

Comment:
- [ ] Mission reservation is similarly not working as intended. When a mission are joined and then a user visits the profile or other page and back, the mission status is cleared from the mission page as well as on the profile.

[mission.webm](https://github.com/jecode93/react-group-project/assets/12745474/f470d710-b4fe-45a5-9bd5-e834c799c5e6)